src/styles/blocks/stars.css

```diff
@@ -0,0 +1,25 @@
+.stars {
+  display: flex;
+  margin-bottom: 0;
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both). Since you have `margin-right` on the `.stars__star` class, it would be better to stick to either top or bottom margins for consistency and to avoid potential margin collapse.